aya/src/obj/mod.rs

```diff
@@ -625,6 +665,15 @@ pub enum ParseError {
 
     #[error("map for section with index {index} not found")]
     MapNotFound { index: usize },
+
+    #[error("map section does not have {num_syms} definitions")]
+    InvalidMapSectionSize { num_syms: usize },
+
+    #[error("the map number {i} in the `maps` section doesn't have a symbol name")]
+    MapSymbolNotFound { i: usize },
```

Comment:
This should actually be `MapSymbolNameNotFound` - we have the symbol, just not a name for it.